src/util/timezones.cpp

```diff
@@ -39,6 +57,13 @@ Timezoner::Timezoner(const char geojson[], std::time_t utc_time_now)
                                     std::to_string(code) + " malformed at offset " +
                                     std::to_string(offset));
     }
+
+    // Run-time check for time zones data: 1500000000 ↔ 'Thu Jul 13 22:40:00 EDT 2017'
+    if (getLocalTimeinfo("EST+5EDT", 1500000000).tm_hour != 22)
```

Comment:
Could the check also just see if the returned time is the same as the given one?